eng/pipelines/templates/steps/vmr-join-verticals.yml

```diff
@@ -0,0 +1,37 @@
+parameters:
+- name: dotNetBuildPass
+  type: string
+  default: final
+
+- name: primaryDependentJob
+  type: string
+  default: Windows_x64
+
+- name: outputFolder
+  type: string
+  default: $(Build.ArtifactStagingDirectory)/artifacts
+
+steps:
+- task: DownloadPipelineArtifact@2
```

Comment:
Does this work? The artifact was published as a build artifact not a pipeline artifact.

Reply:
somehow it works but I updated it to DownloadBuildArtifact